ccdproc/image_collection.py

```diff
@@ -112,6 +112,9 @@ def __init__(self, location=None, keywords=None, info_file=None,
         if keywords:
             self.keywords = keywords
 
+        if extension is not None:
```

Comment:
I am sorry. I didn't actually test what you had written there.
I was getting some other weird errors when testing locally. :-)